_posts/2021-07-15-AI-fairness/AI_fairness_metrics.Rmd

```diff
@@ -0,0 +1,415 @@
+---
+title: "Starting to think about AI Fairness"
+description: > 
+  The topic of AI fairness metrics is as important to society as it is confusing. Confusing it is due to a number of reasons: terminological proliferation, abundance of formulae, and last not least the impression that everyone else seems to know what they're talking about. This text hopes to counteract some of that confusion by starting from a common-sense approach of contrasting two basic positions: On the one hand, the assumption that dataset features may be taken as reflecting the underlying concepts ML practitioners are interested in; on the other, that there inevitably is a gap between concept and measurement, a gap that may be bigger or smaller depending on what is being measured. In contrasting these fundamental views, we bring together concepts from ML, legal science, and political philosophy.
```

Comment:
> Confusing it is

This is a bit of a Yoda-like construct 😉. If you don't want to repeat “It's confusing for a number of reasons…” you could do something like “The confusion arises for a number of reasons” or something like that

Reply:
> In contrasting these fundamental views, we bring together concepts from ML, legal science, and political philosophy.
 
 I think this should be either “In contrast to these fundamental views…” or “Contrasting these fundamental views…”

Reply:
the former would convey a different meaning ... In fact I'm not that happy with the latter either, - I think leaving in the "In" makes more clear what I mean...

... and sorry about the "Yoda-like construct " :-)) ... I need to keep this one, it can't be helped